<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookInfoWindow</class>
 <widget class="QDialog" name="BookInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>660</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>НАЗВАНИЕ</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>660</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>АВТОР</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>660</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ГОД</string>
   </property>
  </widget>
  <widget class="QLabel" name="genreLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>660</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>ЖАНР</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>660</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>ФОТО</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
